spec/asyncapi.md

```diff
@@ -1867,7 +1867,40 @@ Name | Allowed values | Notes
 
 ###### Multi Format Schema Object Example with Avro
 
-<!-- asyncapi-example-tester:{test:'Muti Format Schema Object - Avro',json_path:'$.channels'} -->
+<!-- asyncapi-example-tester:{"test":"Muti Format Schema Object - Avro","json_path":"$.channels"} -->
+
+```json
+"channels":{
+  "example": {
+    "messages": {
+      "myMessage": {
+        "payload": {
+          "schemaFormat": "application/vnd.apache.avro;version=1.9.0",
+          "schema": {
+            "type": "record",
+            "name": "User",
+            "namespace": "com.company",
+            "doc": "User information",
+            "fields": [
+              {
+                "name": "displayName",
+                "type": "string"
+              },
+              {
+                "name": "age",
+                "type": "int"
+              }
+            ]
+          }
+        }
+      }
+    }
+  }
+}
+```
```

Comment:
I've added some missing JSON examples while I was going through the document, PTAL @smoya @fmvilas @dalelane @char0n @GreenRover @derberg

Reply:
I would not add any new example in this PR but rather focus on the main issue that is testing current ones. If you want to add new tests later on, you can create a different PR. Otherwise we might be spending time reviewing the examples rather than what it matters in this case.

Reply:
> I would not add any new example in this PR but rather focus on the main issue that is testing current ones. If you want to add new tests later on, you can create a different PR. Otherwise we might be spending time reviewing the examples rather than what it matters in this case.

@smoya 
I understand. I added these because the markdown comment was given for the YAML version and since I'm only considering the JSON ones for the validation, I thought these would be exempted from the validation therefore made this decision. 

I will revert it for now and will open a new PR addressing the JSON versions for these.

Reply:
@smoya 
resolved through a51107d